.github/workflows/npm-publish.yml

```diff
@@ -23,6 +27,15 @@ jobs:
         with:
           node-version: 18
           cache: 'npm'
+      - name: validate latest changelog entry
+        run: |
+          FIRST_CHANGELOG_LINE=$(head -n 1 CHANGELOG.md)
+          if [[ "$FIRST_CHANGELOG_LINE" =~ ^\#[[:space:]]${RELEASE_VERSION//./\\.}.* ]]; then
```

Comment:
Bash does not allow Spaces in Regex and the Dots in the RLEASE_VERSION env have to been escaped